generated/protocol-tests/tests/input/rest-xml-1/CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 add_project(rest-xml-1-input-protocol-tests
     "Tests for the protocol rest-xml of AWS C++ SDK"
     testing-resources
-    aws-cpp-sdk-rest-xml-protocol-namespace
```

Comment:
should cmake be changing in this revision? some seem to be getting `-namespace` others removing `-namespace` anything up there?

Reply:
It is the issue of the code gen not being stable. Will fix soon.